emukit/benchmarking/loop_benchmarking/metrics.py

```diff
@@ -17,7 +17,8 @@ class MeanSquaredErrorMetric(Metric):
     """
     Mean squared error metric stored in loop state metric dictionary with key "mean_squared_error".
     """
-    def __init__(self, x_test: np.ndarray, y_test: np.ndarray, name: str='mean_squared_error'):
+
+    def __init__(self, x_test: np.ndarray, y_test: np.ndarray, name: str = 'mean_squared_error'):
```

Comment:
actually the one without spaces is what PEP recommends: "Don't use spaces around the = sign when used to indicate a keyword argument"

applies here and below